tests/test_reader.py

```diff
@@ -155,7 +155,7 @@ def test_container_train_property(self):
                       [('train', 'csv'), ('test', 'xls'), ('train_metadata', 'tsv')],
                       [('train', 'jsonlines'), ('test', 'jsonlines')]]
 
-        converter = {'id': str, 'feature1': int, 'feature2': int, 'candidate': str}
+        converter = {'id': str, 'feature1': np.int64, 'feature2': np.int64, 'candidate': str}
```

Comment:
Just curious why these needed to be changed.

Reply:
See comment above.